src/ripple/app/tests/Flow_test.cpp

```diff
@@ -814,7 +814,7 @@ struct Flow_test : public beast::unit_test::suite
         {
             // First pass through a strand redeems, second pass issues, through an offer
             // limiting step is not an endpoint
-            Env env (*this, features(featureFlowV2));
+            Env env (*this, features(featureFlowV2), features(featureOwnerPaysFee));
```

Comment:
I'm wincing as I write this.  We are, for the moment, supporting two operational modes: owner pays fee and recipient pays fee.  But I think as things stand right now only one of those modes is being tested: owner pays fee.  Depending on the longevity of recipient pays fee mode that may be acceptable.  I felt like I should at least bring up the topic.